Sources/MapboxMaps/Annotations/Generated/PolygonAnnotationManager.swift

```diff
@@ -10,18 +10,12 @@ public class PolygonAnnotationManager: AnnotationManager {
     // MARK: - Annotations -
 
     /// The collection of PolygonAnnotations being managed
-    public private(set) var annotations = [PolygonAnnotation]() {
+    public var annotations = [PolygonAnnotation]() {
```

Comment:
Do we want to discourage setting this repeatedly in documentation? (also applies to the other annotation managers)